Foundation/IndexPath.swift

```diff
@@ -10,6 +10,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if os(Android)
+import Glibc // for free()
```

Comment:
I’ll removed this. There is an @_exported import of Glibc in Foundation/NSSwiftRuntime.swift